src/apiCalls.js

```diff
@@ -0,0 +1,25 @@
+const checkForError = (response) => {
+  if(!response.ok) {
+    throw new Error(response.status)
+} else {
+    return response.json()
+}
+}
+
+export const fetchUserName = (userName) => {
+  return fetch("https://warm-scrubland-95764.herokuapp.com/api/v1/sessions", {
+      method: "POST",
+      headers: {"Content-Type": "application/json"},
+      body: JSON.stringify({username: userName})
+    }) 
+    .then(response => checkForError(response))
```

Comment:
I think this can just be `.then(checkForError)`, but I'm not entirely sure that is true.